### PR TITLE
Remove go-build from repo

### PR DESCRIPTION
**Pull request by yiranwang52**
Also removed tools/benchmark.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -205,9 +103,42 @@ devcluster: $(LINUX_BINS) docker_stop
...
+		$(GO) test -bench=. -run=$(TEST_DIRS)
+
+update-golden:
+	$(shell UBER_ENVIRONMENT=test UBER_CONFIG_DIR=`pwd`/config/origin go test ./client/cli/ -update 1>/dev/null)
+	@echo "generated golden files"
+
 # ==== MOCKS ====
 
-mockgen = GOPATH=$(OLDGOPATH) $(GLIDE_EXEC) -g $(GLIDE) -d $(GOPATH)/bin -x github.com/golang/mock/mockgen -- mockgen
+mockgen = $(GLIDE_EXEC) -g $(GLIDE) -d $(GOPATH)/bin -x github.com/golang/mock/mockgen -- mockgen
```
Comment (codygibb) [inline Makefile:141]: `$(GLIDE_EXEC)` no longer exists. You'll need to work out an alternative. Perhaps system glide will work, although I believe we ran into a lot of problems with system glide on Jenkins.

File: Makefile [hunk 2]
```diff
@@ -48,129 +15,60 @@ $(PROTO): $(wildcard proto/*)
...
-	tools/bin/testfs/testfs.linux \
-	tracker/tracker.linux
-
-$(LINUX_BINS):: $(FAUXFILE) $(FAUX_VENDOR)
+	agent/agent \
+	build-index/build-index \
+	origin/origin \
+	proxy/proxy \
+	tools/bin/testfs/testfs \
+	tracker/tracker
```
Comment (codygibb) [inline Makefile:32]: Why are we removing all system binaries?

File: Makefile [hunk 3]
```diff
@@ -205,9 +103,42 @@ devcluster: $(LINUX_BINS) docker_stop
...
+
+LINUX_TOOLS = \
+	tools/bin/reload/reload \
+	tools/bin/simulation/simulation \
+	tools/bin/trackerload/trackerload
+
+tools/bin/reload/reload:: $(wildcard tools/bin/reload/reload/*.go)
+	$(BUILD_LINUX)
+
+tools/bin/simulation/simulation:: $(wildcard ttools/bin/simulation/simulation/*.go)
```
Comment (codygibb) [inline Makefile:116]: Spelling. `s/ttools/tools`

File: Makefile [hunk 4]
```diff
@@ -205,9 +103,42 @@ devcluster: $(LINUX_BINS) docker_stop
 		kraken-devcluster:latest
 	docker logs -f kraken-devcluster
 
+# ==== TOOLS ====
+
+LINUX_TOOLS = \
+	tools/bin/reload/reload \
+	tools/bin/simulation/simulation \
+	tools/bin/trackerload/trackerload
```
Comment (codygibb) [inline Makefile:111]: None of our tools should be restricted to linux, we should be able to run them locally. I think using the system compiler is appropriate.

File: Makefile [hunk 5]
```diff
@@ -205,9 +103,42 @@ devcluster: $(LINUX_BINS) docker_stop
...
+tools: $(LINUX_TOOLS)
+
+# Creates a release summary containing the build revisions of each component
+# for the specified version.
+releases/%:
+	./scripts/release.sh $(subst releases/,,$@)
+
+.PHONY: bench
+bench:
+	$(ECHO_V)cd $(FAUXROOT); $(TEST_ENV)	\
```
Comment (codygibb) [inline Makefile:132]: None of these variables exist anymore.

File: Makefile [hunk 6]
```diff
@@ -205,9 +103,42 @@ devcluster: $(LINUX_BINS) docker_stop
...
+		$(GO) test -bench=. -run=$(TEST_DIRS)
+
+update-golden:
```
Comment (codygibb) [inline Makefile:135]: You can remove this target. We don't have any golden files.

File: .gitignore [hunk 7]
```diff
@@ -28,12 +28,12 @@ tools/bin/pexchange/pexchange
...
-origin/origin.linux
-proxy/proxy.linux
-build-index/build-index.linux
-tools/bin/testfs/testfs.linux
+tracker/tracker
+agent/agent
+origin/origin
+proxy/proxy
+build-index/build-index
+tools/bin/testfs/testfs
```
Comment (codygibb) [inline .gitignore:36]: This is redundant. All of these binaries are declared above.

**Comment by luis9529** [comment]
#2

**Comment by luis9529** [comment]
> [# 2](https://github.com/uber/kraken/pull/2)
> 
> * Bump dependencias
> * Eliminar go-build

**Pull request merged**